bigbluebutton-config/helpers/xml_helper_conf.rb

```diff
@@ -0,0 +1,56 @@
+# BigBlueButton open source conferencing system - http://www.bigbluebutton.org/
+#
+# Copyright (c) 2017 BigBlueButton Inc. and by respective authors (see below).
+#
+# This program is free software; you can redistribute it and/or modify it under
+# the terms of the GNU Lesser General Public License as published by the Free
+# Software Foundation; either version 3.0 of the License, or (at your option)
+# any later version.
+#
+# BigBlueButton is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+# details.
+#
+# You should have received a copy of the GNU Lesser General Public License along
+# with BigBlueButton; if not, see <http://www.gnu.org/licenses/>.
+
+require 'optimist'
```

Comment:
How are we handling the gems for this? The recording system no longer installs gems system-wide which this can use, and I refuse to add any new packaging code that installs gems from within the packaging.

Ideally, either this script could be refactored to use gems available by default (or in ubuntu packages), or we will have to do like in the recording system and include a bundle of vendored gems.